lib/util/cache/package/decorator.ts

```diff
@@ -66,21 +70,51 @@ export function cache<T>({
       return callback();
     }
 
-    const cachedResult = await packageCache.get<unknown>(
+    finalKey = `cache-decorator:${finalKey}`;
```

Comment:
Is the purpose of this line to disregard/ignore/let expire all previously cached versions?

If I understand right, we'll want to wait until v35 which has automatic cleanup of file-based cache. Otherwise in v34 this would double the package cache on disk with no cleanup possible

Reply:
Yes, we need to reset the older caches, or another option is to carefully migrate existing values. However, to me looks like it isn't worth the effort.